ydb/core/persqueue/read_balancer.h

```diff
@@ -254,8 +257,10 @@ class TPersQueueReadBalancer : public TActor<TPersQueueReadBalancer>, public TTa
         }
         RegisterEvents.clear();
 
-        Y_ABORT_UNLESS(0 < AppData(ctx)->PQConfig.GetBalancerWakeupIntervalSec());
-        ctx.Schedule(TDuration::Seconds(AppData(ctx)->PQConfig.GetBalancerWakeupIntervalSec()), new TEvents::TEvWakeup()); //TODO: remove it
+        auto wakeupInterval = AppData(ctx)->PQConfig.GetBalancerWakeupIntervalSec();
+        Y_ABORT_UNLESS(0 < wakeupInterval);
```

Comment:
Remove this ABORT. Incorrect PQConfig will kill cluster. Dangerous! Use Max(wakeupInteval, 1sec)